src/ralph/backends/data/ldp.py

```diff
@@ -0,0 +1,259 @@
+
+    APPLICATION_KEY: str = None
+    APPLICATION_SECRET: str = None
+    CONSUMER_KEY: str = None
+    DEFAULT_STREAM_ID: str = None
+    ENDPOINT: Literal[
+        "ovh-eu",
+        "ovh-us",
+        "ovh-ca",
+        "kimsufi-eu",
+        "kimsufi-ca",
+        "soyoustart-eu",
+        "soyoustart-ca",
+    ] = "ovh-eu"
+    REQUEST_TIMEOUT: int = None
+    SERVICE_NAME: str = None
+
+
+class LDPDataBackend(HistoryMixin, BaseDataBackend):
+    """OVH LDP (Log Data Platform) data backend."""
+
+    name = "ldp"
+    settings_class = LDPDataBackendSettings
+
+    def __init__(self, settings: settings_class = None):
+        """Instantiate the OVH LDP client."""
+        self.settings = settings if settings else self.settings_class()
+        self.service_name = self.settings.SERVICE_NAME
+        self.stream_id = self.settings.DEFAULT_STREAM_ID
+        self.timeout = self.settings.REQUEST_TIMEOUT
+        self._client = None
+
+    @property
+    def client(self):
+        """Create an ovh.Client if it doesn't exists."""
+        if not self._client:
+            self._client = ovh.Client(
+                endpoint=self.settings.ENDPOINT,
+                application_key=self.settings.APPLICATION_KEY,
+                application_secret=self.settings.APPLICATION_SECRET,
+                consumer_key=self.settings.CONSUMER_KEY,
+            )
+        return self._client
+
+    def status(self) -> DataBackendStatus:
+        """Check whether the default service_name is accessible."""
+        try:
+            self.client.get(self._get_archive_endpoint())
+        except ovh.exceptions.APIError as error:
+            logger.error("Failed to connect to the LDP: %s", error)
+            return DataBackendStatus.ERROR
+        except BackendParameterException:
+            return DataBackendStatus.ERROR
+
+        return DataBackendStatus.OK
+
+    def list(
+        self, target: str = None, details: bool = False, new: bool = False
+    ) -> Iterator[Union[str, dict]]:
+        """List archives for a given target stream_id.
+
+        Args:
+            target (str or None): The target stream_id where to list the archives.
+                If target is `None`, the `DEFAULT_STREAM_ID` is used instead.
+            details (bool): Get detailed archive information instead of just ID's.
+            new (bool): Given the history, list only not already read archives.
+
+        Yields:
+            str: If `details` is False.
+            dict: If `details` is True.
+
+        Raises:
+            BackendParameterException: If the `target` is `None` and no
+                `DEFAULT_STREAM_ID` is given.
+            BackendException: If a failure during retrieval of archives list occurs.
+        """
+        list_archives_endpoint = self._get_archive_endpoint(stream_id=target)
+        logger.debug("List archives endpoint: %s", list_archives_endpoint)
+        logger.debug("List archives details: %s", str(details))
```

Comment:
I think it's better to log this info it if the `details` parameter is set to True